### ISSUE TITLE
new term: Susceptibility

### ISSUE THREAD

**Issue by yankeetomer** [obophenotype/human-phenotype-ontology#4310]
While the current terms nested under “Mode of Inheritance” are sufficient for mendelian inheritance, these are not sufficient for the purposes of indexing genotype-phenotype connections that are based on population based evidence, most notably GWAS. While the term “Multifactorial inheritance” (HP:0001426) can be sufficient for some cases, it is inaccurate in others.
Moreover, OMIM has a large number of diseases which are denoted as “Susceptibility to” but this important label can not be incorporated into the HPO terms used for the disease. It should be noted that many HPO terms contain the word “Susceptibility” to describe very specific phenotypes but in my opinion a general term is lacking to attach to gene-disease connections.

Therefore, I suggest a new term to be added under mode of inheritance:

### 1. Preferred term label

- Susceptibility

### 2. Synonyms
-Association

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

**Susceptibility**
An inherited increase in the risk of developing an acquired disease.

PMID: 16932734
https://www.cancer.gov/publications/dictionaries/cancer-terms/def/genetic-susceptibility

PMID: 28686856
PMID: 23633916
PMID: 28067912 

### 4. Parent term
Mode of inheritance
HP:0000005
(http://compbio.charite.de/hpoweb/showterm?id=HP:0000005)

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

Autism, Susceptibility To, 3 
OMIM:608049
Alzheimer Disease, Susceptibility To, Mitochondrial 
OMIM:502500
Bleeding Disorder, Platelet-type, 13, Susceptibility To 
OMIM:614009
Aortic Aneurysm, Familial Thoracic 11, Susceptibility To 
OMIM:617349

### 6. Your nano-attribution 

Tomer Talmy M.D.
Organization: Emedgene Technologies
ORCID: https://orcid.org/0000-0002-7440-8864

**Comment by pnrobinson** [comment]
It seems that the term Polygenic Inheritance covers this
https://hpo.jax.org/app/browse/term/HP:0010982
The problem with a term named "Susceptibility" is that it overlaps with many of the existing terms such as this and thus curators would have a choice of which term to annotate to -- and if different databases use different terms, then it would be hard to interoperate/exchange.
Can you use Polygenic Inheritance? If not, please elaborate the reasons!

**Comment by yankeetomer** [comment]
I think my issue is more with situations in which a monogenic cause is believed to increase susceptibility to a certain condition, and not necessarily the scenario of polygenic inheritance. I agree that the term appears frequently in association with certain conditions (e.g Susceptibility to infections) but these specific terms may not be able to encompass the vast spectrum of possible increased susceptibility believed to arise as a result of a monogenic variant.
Hope that makes sense

**Comment by pnrobinson** [comment]
The thing is that we try to not have terms with partially overlapping meanings.
All of the susceptibility terms in OMIM (like the ones you mentioned above) can correctly be annotated to Polygenic.
There are monogenic diseases such as breast carcinoma associated with BRCA1 mutations. It is clear that the inheritance is autosomal dominant, but what is being inherited is the susceptibility to get breast cancer. We would annotate this by adding the frequency of carriers that get breast cancer in studies. The new HPO annotation format also allows one to indicate the affected sex.

So I think adding a new term called susceptibility would be confusing in that it would partially overlap with both polygenic and some of the mendelian inheritance terms. I think that the susceptibility you are talking about is best annotated with frequencies, e.g., 4/8 patients with disease X got Persistent EBV viremia instead of saying that disease X inheritance is "susceptibility".

**Comment by yankeetomer** [comment]
Understood, I will indeed try to make the indexing work using frequencies and I hope that will suffice. Thanks a lot for the detailed explanation, Tomer

**Comment by pnrobinson** [comment]
Sure -- consider contributing annotations to the public data set too if you like. I will closes this issue for now.
